lib/matplotlib/tests/test_image.py

```diff
@@ -855,8 +939,10 @@ def test_imshow_masked_interpolation():
     data = np.ma.masked_array(data, mask)
 
     fig, ax_grid = plt.subplots(3, 6)
+    interps = mimage._interpd_
+    interps.pop('antialiased', None)
```

Comment:
you need to operate on a copy, not on the global dict itself
also the second argument (None) serves no purpose here and can be removed

Reply:
Gah!

Reply:
Oh, but hang on, this isn't the failing test....

Reply:
OK< that fixed it.  I had no idea that a global was global across test files.